.github/workflows/cross-compile.yml

```diff
@@ -145,12 +151,13 @@ jobs:
         if: ${{ matrix.sdk-version == 'MacOSX10.9.sdk' }}
         run: |
           wget -qO- https://github.com/phracker/MacOSX-SDKs/releases/download/11.3/MacOSX10.9.sdk.tar.xz \
-            | tar -xjv -C /Applications/Xcode_12.2.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs
+            | tar -xjv -C $XCODE_12_DEVELOPER_DIR/Platforms/MacOSX.platform/Developer/SDKs
```

Comment:
Shouldn't this be `DEVELOPER_DIR` (instead of `XCODE_12_DEVELOPER_DIR`)?

Reply:
On Github CI, that one is not defined. We have:
```
XCODE_11_DEVELOPER_DIR=/Applications/Xcode_11.7.app/Contents/Developer
XCODE_12_DEVELOPER_DIR=/Applications/Xcode_12.5.1.app/Contents/Developer
XCODE_13_DEVELOPER_DIR=/Applications/Xcode_13.2.1.app/Contents/Developer
```
and I assume they patch the minor version regularly, as literally defined `/Applications/Xcode_12.2.app` before this change was not found.

Reply:
Thanks.  Sorry, I had missed the fact that this PR removes the old environment variable.

```diff
-    env:
-      DEVELOPER_DIR: /Applications/Xcode_12.2.app/Contents/Developer
```